src/WorkItemMigrator/JiraExport/JiraProvider.cs

```diff
@@ -401,9 +401,30 @@ public string GetUserEmail(string usernameOrAccountId)
             }
         }
 
```

Comment:
The GetCustomId method is now just a wrapper around the new GetCustomIdList() which returns all of the possible matches, rather than just the first.  With this in place, no existing code should be effected.